README.md

```diff
@@ -803,16 +803,16 @@ As you can see, it looks like if we set the `LOG_DEBUG` environment variable,
 we'll get some debug logging. So let's use `balena env` to do this:
```

Comment:
```suggestion
we'll get some debug logging (If you don't see this message ensure you are pushing the code from this masterclass, and not `balena-cli-masterclass`!). So let's use `balena env` to do this:
```

Reply:
Adding this change from https://github.com/balena-io/balena-cli-advanced-masterclass/pull/30/files